conda/recipes/cudf/meta.yaml

```diff
@@ -35,7 +35,7 @@ requirements:
     - protobuf
     - python
     - pandas >=1.0,<1.2.0dev0
-    - cupy >7.1.0,<9.0.0a0
+    - cupy >=7.8.0,<9.0.0a0
```

Comment:
Why do we need to restrict it to `>=7.8.0` when cuDF doesn't use any newer features of CuPy?